src/StatusQ/Popups/StatusSearchPopup.qml

```diff
@@ -18,19 +18,33 @@ StatusModal {
     showFooter: false
 
     property string searchText: contentComponent.searchText
-    property string noResultsLabel: "No results"
+    property string noResultsLabel: qsTr("No results")
+    property string firstItemText: qsTr("Anywhere")
```

Comment:
How about calling this `defaultSearchLocationText`?

Reply:
Also, no need to `qsTr()` them here because it's the applications reponsibility to i18n app messages.